R/JPSEDF.R

```diff
@@ -1,82 +1,102 @@
 #' Computes the estimator and variance for each individual ranker
 #'
-#' @param RV Rank values for Y.
+#' @param ranks Ranks of Y.
 #' @param Y Response measurements.
-#' @param set_size Set Size.
+#' @param set_size Set size for each raking group.
 #' @param N Finite population size.
 #' @param coef Coefficients used in variance computation when sample size is n.
 #' @param coef_del Coefficients used in variance computation when the i-th unit is deleted.
 #' @param replace Logical. Sample with replacement?
-#' @param model If model is 0, it's design based inference, if model = 1, it is model based inference using super population model.
-#' @param K
+#' @param model An inference mode:
+#' - `0`: design based inference
```

Comment:
I wonder if this should just be changed to a boolean (e.g. model_based = TRUE or model_based = FALSE)? This would more closely align with standard R style.

Reply:
agree, will update accordingly

Reply:
Same thing occurs in several other locations. I think I commented on one more, but there are more that I didn't comment on.